internal/service/alertconfiguration/resource_alert_configuration_test.go

```diff
@@ -20,27 +21,32 @@ const (
 )
 
 func TestAccConfigRSAlertConfiguration_basic(t *testing.T) {
+	proxyPort, teardown := replay.SetupReplayProxy(t)
+	t.Cleanup(func() { teardown(t) })
```

Comment:
you can make SetupReplayProxy to directly call Cleanup as it has t, so no need to pass or call teardown from outside

Reply:
great point, adjusted.